test/fhirtypes/ElementDefinition.assignString.test.ts

```diff
@@ -339,6 +339,7 @@ describe('ElementDefinition', () => {
       );
     });
 
+    // MAYBE?
```

Comment:
That's... a good question. Should assigning a string be allowed if it corresponds to a Canonical URL of the appropriate type? Or more likely, from the other direction: should it be allowed as long as it's a valid URI, and if we find something with that url, it has the appropriate type? It's extra steps to check, but it would nice to allow it, particularly for cases when it's the canonical of something that is not in SUSHI's scope.

Reply:
TBH, I don't remember why I wrote this comment -- and I suspect I left it in here by mistake!

I think I'm fine w/ keeping behavior as it is. We have traditionally allowed string-based assignment to references as a back-door way to force a specific reference URL without checking it. I think canonical should probably be the same. If you want extra type-checking, using `Canonical(...)`, otherwise use a string to force whatever URL value you want.

Reply:
But I am going to delete this comment.